### PR TITLE
[opt] curvefs: Change the confusing configuration name

### PR DESCRIPTION
**Pull request by 201341**
### What problem does this PR solve?

Issue Number: #2597 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by 201341** [comment]
cicheck

**Comment by 201341** [comment]
cicheck

**Comment by caoxianfei1** [comment]
you can refer the exist pr #2647

**Comment by wuhongsong** [comment]
@Ziy1-Tan  PTAL？

**Review by Ziy1-Tan — APPROVED** [review]

**Comment by 201341** [comment]
@wuhongsong PTAL？

**Review by wuhongsong — APPROVED** [review]

File: curvefs/src/metaserver/metaserver.cpp [hunk 1]
```diff
@@ -136,8 +136,8 @@ void Metaserver::InitLocalFileSystem() {
 void InitS3Option(const std::shared_ptr<Configuration>& conf,
                   S3ClientAdaptorOption* s3Opt) {
     LOG_IF(FATAL, !conf->GetUInt64Value("s3.batchsize", &s3Opt->batchSize));
-    LOG_IF(FATAL, !conf->GetBoolValue("s3.enableDeleteObjects",
-                                      &s3Opt->enableDeleteObjects));
+    LOG_IF(FATAL, !conf->GetBoolValue("s3.enableBatchDelete",
```
Comment (wuhongsong) [inline curvefs/src/metaserver/metaserver.cpp:139]: need forward compatibility, can you fix it later?
https://github.com/opencurve/curve/issues/2671

**Pull request merged**